File: src/main/java/com/tobe/healthy/schedule/repository/TrainerScheduleInfoRepository.java
package com.tobe.healthy.schedule.repository;

import org.springframework.data.jpa.repository.JpaRepository;

import com.tobe.healthy.schedule.domain.TrainerScheduleInfo;

public interface TrainerScheduleInfoRepository extends JpaRepository<TrainerScheduleInfo, Long> {
	TrainerScheduleInfo findOneByTrainerId(Long trainerId);
}

File: src/main/kotlin/com/tobe/healthy/schedule/repository/TrainerScheduleInfoRepository.kt
package com.tobe.healthy.schedule.repository

import com.tobe.healthy.schedule.domain.entity.TrainerScheduleInfo
import org.springframework.data.jpa.repository.JpaRepository

interface TrainerScheduleInfoRepository : JpaRepository<TrainerScheduleInfo, Long> {
    fun findOneByTrainerId(trainerId: Long): TrainerScheduleInfo?
}
